sympy/functions/combinatorial/tests/test_comb_factorials.py

```diff
@@ -268,6 +269,9 @@ def test_binomial():
     assert expand_func(binomial(n, n - 3)) == n*(n - 2)*(n - 1)/6
 
     assert binomial(n, k).is_integer
+    nt = Symbol('nt', integer=False)
+    assert binomial(nt, kp).is_integer is False
```

Comment:
This is valid if kp = 1, but not if kp > 1. For example, ``binomial((1 + sqrt(33))/2, 2) == 4`` by the more general definition. But if we know that nt is rational, then this would be still valid.